proposals/NNNN-feature-profiles.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "[NNNN] - Feature Profiles"
+params:
+  authors:
+    - llvm-beanz: Chris Bieneman
+  status: Under Consideration
+  sponsors:
+    - llvm-beanz: Chris Bieneman
+---
+
+## Introduction
+
+This proposal seeks to introduce a new more formalized way of specifying feature
+profiles as a set of features to allow the compiler to utilize when generating
+outputs. This proposal pairs stricter availability checks with explicit opt-in
```

Comment:
"Feature Profiles" is a new concept being added here?  The current first sentence feels a bit long and clunky.  Here's a stab at another way of wording it:

```suggestion
This proposal introduces feature profiles: a more formalized way to specify a set of features that the compiler can utilize when generating outputs. This proposal pairs stricter availability checks with explicit opt-in
```